### PR TITLE
Fixes #10211-Added hammer cli foreman docker to the gemspec

### PR DESCRIPTION
**Pull request by parthaa**

### CODE REVIEW

File: deploy/katello.spec [hunk 1]
```diff
@@ -116,8 +116,9 @@ Summary:    Common runtime components of %{name}
 Requires:       wget
 Requires:       curl
 Requires:       %{?scl_prefix}rubygem-katello
-Requires:       rubygem-hammer_cli 
+Requires:       rubygem-hammer_cli
 Requires:       rubygem-hammer_cli_foreman
+Requires:       rubygem-hammer_cli_foreman_docker
```
Comment (ehelms) [inline deploy/katello.spec]: I don't think SAM gets docker so this may need to go in the katello RPM section. @thomasmckay

**Comment by ehelms** [comment]
Nitpick: commit message, you are adding this to the RPM spec

File: deploy/katello.spec (hunk 1 above)
Comment (thomasmckay) [inline deploy/katello.spec]: Unfortunately, I think hammer-cli-katello now relies on hammer-cli-foreman-docker. Certainly the docker plugin should not be shipped w/ SAM, but the cli could be and then disabled by SAM install.

https://github.com/Katello/hammer-cli-katello/blob/master/hammer_cli_katello.gemspec#L42

Comment (ehelms) [inline deploy/katello.spec]: If the katello cli requires it then this is unneeded.

Comment (ehelms) [inline deploy/katello.spec]: Looks like this is handled here -- https://github.com/Katello/hammer-cli-katello/blob/master/rubygem-hammer_cli_katello.spec#L27

Comment (ehelms) [inline deploy/katello.spec]: @parthaa so based on the above, it doesn't look like this is needed?

**Comment by parthaa** [comment]
closing this, since hammer_cli_katello is already including this.

**Pull request closed without merging**